packages/core/upload/server/controllers/validation/admin/folder-file.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const { yup, validateYupSchema } = require('@strapi/utils');
+
+const validateDeleteManyFoldersFilesSchema = yup
+  .object()
+  .shape({
+    fileIds: yup
+      .array()
+      .min(1)
+      .of(yup.strapiID().required()),
```

Comment:
The above comment would probably also have an effect here ... I'd prefer to be able to omit it or send an empty array.